tests/velero/restore.go

```diff
@@ -120,6 +117,7 @@ func (c *ClientSet) CreateRestore(ns, targetedNs, backup string) (v1.RestorePhas
 			IncludedNamespaces: []string{ns},
 			RestorePVs:         &snapVolume,
 			BackupName:         backup,
+			ScheduleName:       schedule,
```

Comment:
do we need this?

Reply:
yes, for test case [**_Restore from scheduled backup to different Namespace_**](https://github.com/openebs/velero-plugin/pull/99/files/7553cc6bb4ad69ea4d67c17652d9689dd97f9005#diff-60de5209d5a0ca5555d0bf45dafe12cf0f7717e517339654e44cb55f4199e9baR166)

Reply:
not able to see the changes where we are using variable ScheduleName in this PR?

Reply:
yes. https://github.com/openebs/velero-plugin/blob/7553cc6bb4ad69ea4d67c17652d9689dd97f9005/tests/sanity/backup_test.go#L166